tests/gnrc_tcp/tests/05-garbage-pkts.py

```diff
@@ -96,14 +97,25 @@ def test_short_header(child, src_if, src_ll, dst_if, dst_l2, dst_ll, dst_port):
 @testfunc
 def test_send_ack_instead_of_syn(child, src_if, src_ll,
                                  dst_if, dst_l2, dst_ll, dst_port):
+    # check if entering _event_loop in `@testfunc` decorator generates debug
+    # output
+    debug = child.expect([pexpect.TIMEOUT, 'GNRC_TCP'], timeout=1)
```

Comment:
I would at least add some more stuff from the debug messages to be more specific. Just checking `GNRC_TCP` in a test called `gnrc_tcp` could lead to future confusion ;-). Also the comment above needs to be adopted for whatever you are checking.

Reply:
Hm 'GNRC_TCP: Enter' then? It should be match to all calls to TCP_DEBUG_ENTER

Reply:
How about using a regex that matches the output of TCP_DEBUG_ENTER then?

```suggestion
    debug = child.expect([pexpect.TIMEOUT,
                          r'GNRC_TCP: Enter "\S+", File: .+(\d+)\s"'], timeout=1)
```

Reply:
I've added it.